packages/scripts/package.json

```diff
@@ -0,0 +1,38 @@
+{
+	"name": "@wordpress/scripts",
+	"version": "1.0.1-0",
```

Comment:
I will publish 1.0.0 soon and stick with alpha in the future. I'm still learning how to use Lerna. Thanks for a good hint :)

Reply:
via https://github.com/lerna/lerna#--cd-version

> _"If you have any packages with a prerelease version number (e.g. `2.0.0-beta.3`) and you run `lerna publish` with `--cd-version` and a non-prerelease version increment (major / minor / patch), it will publish those packages in addition to the packages that have changed since the last release."_

That should work for Lerna and `1.0.0-alpha.0` by my reading

Reply:
How about using the `--canary` command? That's more granular and being used by other popular projects like CRA and I am using it on CGB `create-guten-block`.

`lerna publish --independent --npm-client=npm --canary`

Reply:
We have this in `npm run publish:dev`:
https://github.com/WordPress/packages/blob/master/package.json#L36

We might want to revisit how it's configured to simplify the process.